counter-script/project/contracts/counter/Forc.lock

```diff
@@ -0,0 +1,21 @@
+[[package]]
+name = 'contract_abi'
+source = 'path+from-root-4F344E6D2A50F28B'
+dependencies = ['std']
+
+[[package]]
+name = 'core'
+source = 'path+from-root-D62A8C57DD8D2801'
+
+[[package]]
+name = 'counter_contract'
+source = 'member'
+dependencies = [
+    'contract_abi',
+    'std',
+]
+
+[[package]]
+name = 'std'
+source = 'git+https://github.com/fuellabs/sway?tag=v0.40.1#48104d0bde0d343154a5bc39a310092532883235'
```

Comment:
Something is wrong, why is this using an older version of sway?

Reply:
Looking at the other apps, there shouldn't be `Forc.lock` files outside the project folder

Reply:
These just need to be removed